src/features/common/Layout/Navbar/index.tsx

```diff
@@ -125,7 +125,7 @@ export default function NavbarComponent(props: any) {
                     <UserProfileIcon />
                   </div>
                   <p className={router.pathname === SingleLink.onclick ? 'active_icon' : ''}>
-                    {t('common:' + SingleLink.title)}
+                  {isAuthenticated && userInfo ? t('common:' + SingleLink.loggedInTitle) : t('common:' + SingleLink.title)} 
```

Comment:
If there are tenants allowing a login which do not have this resource, this could fail. So we better also check if `SingleLink.loggedInTitle` is defined before using it.

Reply:
added a safety check